src/constructor.cpp

```diff
@@ -2,8 +2,10 @@
 #include "constructor_helpers.hpp"
 #include "kml.h"
 
+// Vector to store pairs of reservoirs
 vector<vector<Pair>> pairs;
```

Comment:
In general this repository attempts to follow self documenting code practices (https://en.wikipedia.org/wiki/Self-documenting_code) - for example in this case it would likely be better to just rename the variable rather than add an additional comment line
```vector<vector<Pair>> reservoir_pairs;```